### ISSUE TITLE
Feature request: Recently used categories

### ISSUE THREAD

**Issue by justcallmelarry** [spliit-app/spliit#99]
Could there be a "recently used categories" when adding a new expense, like the last 5 used categories in order at the top when choosing a category?

I have basically 2-3 categories that I use almost all the time, so would be nice to have them easily reachable.

**Comment by shynst** [comment]
Another idea would be to make categories configurable (I did this hardcoded in my instance in the database, since I was not happy with the predefined ones).

Maybe even use different categories per group - i.e. make them a per group setting?

**Comment by justcallmelarry** [comment]
> Another idea would be to make categories configurable (I did this hardcoded in my instance in the database, since I was not happy with the predefined ones).
> 
> Maybe even use different categories per group - i.e. make them a per group setting?

Both of these seem like separate idea's to the one I'm requesting 😄 And could work in tandem, but I think they should probably be separate issues

**Comment by shynst** [comment]
> > Another idea would be to make categories configurable (I did this hardcoded in my instance in the database, since I was not happy with the predefined ones).
> > Maybe even use different categories per group - i.e. make them a per group setting?
> 
> Both of these seem like separate idea's to the one I'm requesting 😄 And could work in tandem, but I think they should probably be separate issues

You are right here. Actually all three of them could work together. I guess this could be called a tridem then... :)
